agent/src/beerocks/fronthaul_manager/ap_manager/ap_manager_thread.cpp

```diff
@@ -158,9 +158,34 @@ void ap_manager_thread::ap_manager_fsm()
 {
     switch (m_state) {
     case eApManagerState::INIT: {
+        if (!slave_socket) {
+            return;
```

Comment:
Shouldn't there be a warning log here?

Reply:
Not critical. If there is an error when opening the socket, it will be printed on the connect function.